cpp/include/cudf/utilities/bit.hpp

```diff
@@ -79,6 +93,7 @@ constexpr CUDF_HOST_DEVICE inline size_type intra_word_index(size_type bit_index
  *
  * @param bitmask The bitmask containing the bit to set
  * @param bit_index Index of the bit to set
+ * @return void
```

Comment:
Is this needed?

Reply:
This was a bug in old doxygen version `1.8.18`. Version 1.8.20 (which we use) does not throw this error.

Reply:
```suggestion
```